packages/@react-spectrum/combobox/docs/ComboBox.mdx

```diff
@@ -996,7 +996,7 @@ isn't sufficient when resolving issues in your own test cases.
 
 ### Test utils <VersionBadge version="beta" style={{marginLeft: 4, verticalAlign: 'bottom'}} />
 
-`@react-spectrum/test-utils` offers common combobox interaction utilities which you may find helpful when writing tests. See [here](../react-aria/testing.html#react-aria-test-utils) for more information on how to setup these utilities
+`@react-spectrum/test-utils` offers common combobox interaction utilities which you may find helpful when writing tests. See [here](./testing.html#react-spectrum-test-utils) for more information on how to setup these utilities
```

Comment:
won't this need to a be an absolute url?

Reply:
oh does it? I always forget the difference between the built docs and the deployed ones, lemme dig

Reply:
the previous url works on prod right now so I think this is ok. Just to clarify though, this is the old docs, I've updated both new and old docs in this PR